compiler/src/dmd/backend/debugprint.d

```diff
@@ -331,7 +331,7 @@ void WRdefnod()
 @trusted
 const(char)* fl_str(FL fl)
 {
-    __gshared const char*[FLMAX] fls =
```

Comment:
please split this trivial change into a separate PR

Reply:
https://github.com/dlang/dmd/pull/16665

Reply:
Thanks, Nick!